drop-boxes/register-fastq-dropbox/README.md

```diff
@@ -0,0 +1,33 @@
+#QBiC-register-fastq-dropbox
+
+##Resulting data model in openBIS
+Q_TEST_SAMPLE -> Q_NGS_RAW_DATA (with sample code) -> DataSet (directory with files contained)
+
+##Description
+For paired-end sequencing reads in FASTQ format, the file structure needs to look like this:
+
+```
+<QBIC sample code>.fastq // Directory
+    |-- <QBIC sample code>_R1.fastq
+    |-- <QBIC sample code>_R1.fastq.sha256sum
+    |-- <QBIC sample code>_R2.fastq
+    |-- <QBIC sample code>_R2.fastq.sha256sum
+```
```

Comment:
How can I verify this structure from the code?

Reply:
https://github.com/qbicsoftware/etl-scripts/blob/master/drop-boxes/register-fastq-dropbox/register-fastq-dropbox.py#L123 ff are probably best.
The folder is registered, contained files which contain ".testorig" and ".origlabfilename" are removed (nothing is done with this metadata, but it is how it is at the moment) and everything else is registered. However, as you can see the .sha256sums are not necessary for the script to work. It's just that they are automatically created by the datahandler. and are thus part of the directory. Shall I remove that part, as it's a specific implementation for QBiC that does not have anything to do with the ETL script itself?

Reply:
But how come R1 and R2 into play? Does it matter for the script (didn't find a line where this was required)? Or should it just display that both strands should be provided?

Reply:
The latter, the script does not care :)